src/components/loadable-item.cjsx

```diff
@@ -18,9 +18,25 @@ module.exports = React.createClass
     renderItem: React.PropTypes.func.isRequired
     saved: React.PropTypes.func
     load: React.PropTypes.func
+    renderLoading: React.PropTypes.func
+    update: React.PropTypes.func
+
+  componentDidMount: -> @reload({})
+  componentDidUpdate: (oldProps) -> @reload(oldProps)
```

Comment:
I was hoping to include https://github.com/openstax/tutor-js/compare/reload-when-mounted...reload-when-mounted-2 next. That way the user does not see the `Loading...` every time they visit the calendar and we can take advantage of React's core feature of only updating the DOM nodes that changed (so once the new data is loaded the screen just updates)